scanner/matcher/matcher.go

```diff
@@ -160,3 +161,18 @@ func (m *matcherImpl) Close(ctx context.Context) error {
 	m.pool.Close()
 	return err
 }
+
+// Ready return true if the matcher is ready.
+func (m *matcherImpl) Ready(ctx context.Context) error {
+	ok, err := m.libVuln.Initialized(ctx)
```

Comment:
I think this just means there is at least one row in the `vuln` table. Is that what you were expecting?

Reply:
Yes. There is "something" in the vulns, like noticing the updater is kicking in or offline mode vulns being provided to Central. The second scenario is debatable, and we may want to keep the POD ready even if there is no offline bundle in Central.